src/processTargets.ts

```diff
@@ -106,6 +128,28 @@ function processSingleRangeTarget(
   });
 }
 
+function getRangePosition(
+  target: TypedSelection,
+  isStart: boolean,
+  exclude: boolean
+) {
```

Comment:
This one could use a rename + docstring.  Not clear to me what it does / what params mean